### DISCUSSION TITLE
Unable to locate installed game.

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [lutris/lutris#4986] [Q&A] [answered]
I just installed Valorant using Lutris's GUI

![image](https://github.com/lutris/lutris/assets/42837531/2661dc3d-ee16-464a-8d87-8133ccde8356)

and then restarted my computer. Lutris doesn't list it as installed, and there's no `.desktop` file in applications:/ despite me requesting it.

**Comment by roke-julian-lockhart** [comment]
This is problematic mostly because I don't know how to cleanly uninstall the game now.

Is

```pwsh
Remove-Item -LiteralPath "$HOME/Games/valorant"
```

fine? I don't know whether the system registry is virtualized for each installation.

**Comment by danieljohnson2** [comment, marked as answer]
Lutris normally installs games in it's Games directory; that would normally be something like what you gave- though does $HOME even work in PowerShell? I thought it was $env:HOME in PowerShell. Maybe that's why it's not working for you.

The game should be entirely in that directory, except for the .desktop files Lutris creates for your desktop or menu, if you select those options. I think the Lutris 'Remove' command removes those too, but other than that you can just delete the directory.

Each game install (for Wine) gets its own "prefix", which means it gets its own registries and its own copies of any Windows software you install there. It's not a VM or a sandbox, but there's some isolation there.

However, the game *itself* can spew files all over your home directory. Some do. Lutris can't fix that, though you can use the misnamed "Sandbox" feature in the game configuration to redirect some of that to a place  you chose. This doesn't really deliver security, despite the name, but it can control the mess.

**Reply by roke-julian-lockhart** [reply]
@danieljohnson2,

> Lutris normally installs games in it's Games directory; that would normally be something like what you gave- though does $HOME even work in PowerShell? I thought it was $env:HOME in PowerShell.

`$Env:HOME` is indeed the correct way to refer to the relevant POSIX environment variable, but `$HOME` is like `$_` in that it's a shell-specific way of referring in a cross-platform manner to the user-specific home directory. It's a slightly more high-level abstraction masquerading as a mere variable.

> Maybe that's why it's not working for you.

Actually, the command does work. I just wanted to know whether

> Each game install (for Wine) gets its own "prefix", which means it gets its own registries and its own copies of any Windows software you install there. It's not a VM or a sandbox, but there's some isolation there.

...so I'll delete it, then. Thanks.

**Reply by danieljohnson2** [reply]
I quite dislike PowerShell, and this why. It's a mass of special rules, and I can never guess what it'll do next.

Ah, well. Delete away. I think you can close this discussion, unless you've got more questions.

**Comment by strycore** [comment]
Please don't install Valorant on Lutris, it has no chance to run.

**Reply by roke-julian-lockhart** [reply]
Thanks. I do know that. I was just testing the installation process.
